conf/application.conf

```diff
@@ -294,8 +294,9 @@ play.filters {
     permittedCrossDomainPolicies = "master-only"
 
     # The Content-Security-Policy header. If null, the header is not set.
-    #contentSecurityPolicy = "default-src 'self'; script-src 'self'; object-src 'none'; style-src 'self' https://fonts.googleapis.com https://fonts.gstatic.com; img-src 'self'; media-src 'none'; font-src https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self'; child-src 'none'; form-action 'none'; frame-ancestors 'none'; report-uri 'none'"
-    contentSecurityPolicy = null // FIXME #15 enable security policy
+    # FIXME WARNING unsafe-inline SHOULD be disabled, but currently webpack loads styles inline so we need it
+    # FIXME WARNING unsafe-eval SHOULD be disabled, but our current Vue build uses it to compile templates (=> precompile them to render functions)
+    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval'; object-src 'none'; style-src 'self' https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline'; img-src 'self'; media-src 'none'; font-src https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' wss://localhost:9443; child-src 'none'; form-action 'self'; frame-ancestors 'none'; report-uri 'none';"

```

Comment:
Does `connect-src 'self' wss://localhost:9443` works even on our deployment server which is e.g. http://learn-duel.herokuapp.com/?

Reply:
nope, it doesn't. should I change it to accept connections from `learn-duel.herokuapp.com` as well?

Reply:
yes and also `learn-duel-dev.herokuapp.com`

Reply:
done, the wildcard should work but we have to test it once it is on the develop branch and deployed